Source/RapyutaSimulationPlugins/Private/Robots/RRBaseRobot.cpp

```diff
@@ -150,7 +150,7 @@ void ARRBaseRobot::PreInitializeComponents()
 {
     if (ROSSpawnParameters)
     {
-        InitPropertiesFromJSON();
+        ensure(InitPropertiesFromJSON());

```

Comment:
ensure will stop sim if it uses debugger.
I think it is too much to stop sim with parameter parsing.

Reply:
oh really, I suppose ensure only print the callstack & only verify/check will stop sim, ok so here not so essential since error is already logged in the function